drodrpg/DRODLib/CurrentGame.cpp

```diff
@@ -6871,7 +6888,9 @@ void CCurrentGame::ProcessPlayerMoveInteraction(int dx, int dy, CCueEvents& CueE
 		//Level map.
 		//Mark all non-secret rooms in level on map.
 		CIDSet roomsInLevel = CDb::getRoomsInLevel(this->pLevel->dwLevelID);
+		CIDSet roomsWithStatesToUpdate = CIDSet();
```

Comment:
You can omit the "= CIDSet()" -- this is a default constructor option, so it's automatically called implicitly.